### PR TITLE
Add warp docs

### LINKED ISSUE #337: Document `StartWarp` Cheatcodes

### LINKED ISSUE #342: Document `stop_*` cheatcodes
- [x] stop_roll
- [ ] stop_warp
- [ ] stop_prank

### PR DESCRIPTION
**Pull request by cptartur**
Closes #337 
Closes #342

## Introduced changes

- Added `start_warp` docs
- Added `stop_warp` docs

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code

### CODE REVIEW

File: docs/src/appendix/cheatcodes/stop_warp.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# `stop_warp`
+
+> `fn stop_warp(contract_address: ContractAddress)`
+
+Cancels the [`start_warp`](./stop_warp.md) for the contract at the given address.
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/stop_warp.md:5]:
```suggestion
Cancels the [`start_warp`](./start_warp.md) for the contract at the given address.
```

**Review by war-in — APPROVED** [review]
I guess PR should be named `Add warp, roll and prank docs` 😅

**Comment by war-in** [comment]
Oh, I can see now. It just has to be rebased

File: docs/src/appendix/cheatcodes/stop_prank.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# `stop_prank`
+
+> `fn stop_prank(contract_address: ContractAddress)`
+
+Cancels the [`stop_prank`](./stop_prank.md) for the contract at the given address.
```
Comment (Radinyn) [inline docs/src/appendix/cheatcodes/stop_prank.md:5]:
```suggestion
Cancels the [`start_prank`](./start_prank.md) for the contract at the given address.
```

File: docs/src/appendix/cheatcodes/start_prank.md [hunk 3]
```diff
@@ -2,18 +2,33 @@
 
 > `fn start_prank(contract_address: ContractAddress, caller_address: ContractAddress)`
 
-Changes the caller address in the `ExecutionInfo` struct and methods utilizing it (e.g. `get_caller_address`).
+Changes the caller address for a contract at the given address.
+This change can be canceled with [`stop_prank](./stop_prank.md).
```
Comment (Radinyn) [inline docs/src/appendix/cheatcodes/start_prank.md:6]:
```suggestion
This change can be canceled with [`stop_prank`](./stop_prank.md).
```

**Review by piotmag769 — APPROVED** [review]

File: docs/src/appendix/cheatcodes/start_warp.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```rust
+// ...
+#[external(v0)]
+impl IContractImpl of IContract<ContractState> {
+    #[storage]
+    struct Storage {
+        // ...
+
+        stored_block_timestamp: u64
+    }
```
Comment (war-in) [inline docs/src/appendix/cheatcodes/start_warp.md:22]: I didn't know we can define storage inside impl :o

**Pull request merged**